core/conn/jdbc_type2/native/SqlInterface.cpp

```diff
@@ -2570,12 +2570,9 @@ SQLRETURN ALLOCSQLMXHDLS_SPJRS(SRVR_STMT_HDL *pSrvrStmt, SQLSTMT_ID *callpStmt,
 	if (pModule->module_name == NULL)
 	{
 		DEBUG_OUT(DEBUG_LEVEL_STMT,("***pModule->module_name == NULL  Call AllocStmtForRs()"));
-/* Commenting out for now - will be looked at when SPJ is supported
- #ifdef NSK_PLATFORM
 		CLI_AllocStmtForRS(callpStmt,
```

Comment:
@selvaganesang For spj tests, a spj jar package is required while create a library. I'm not sure the path of the JAR. it seems to be a little complicated. The JAR should be existing on server-side(every node). i'm not sure the IP address/host name(where to upload) while the tests running.